code/game/machinery/door_control.dm

```diff
@@ -5,7 +5,7 @@
 	icon_state = "doorctrl0"
 	desc = "A remote control-switch for a door."
 	power_channel = ENVIRON
-	var/id_tag = null
+	 
```

Comment:
I'm going to stop commenting on all of these individually because I'm sure it's spamming the conversation page but you can find the rest